### DISCUSSION TITLE
Welcome to xabber-web Discussions!

### DISCUSSION THREAD

**Discussion by andrewnenakhov** [redsolution/xabber-web#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andrewnenakhov** [comment]
Just trying out this new discussion thing.

**Comment by germansfedcenko** [comment]
any updates for Xabber....?
it seems like it was updated many years ago.....
App disappeared from Google Play at all.....

**Reply by andrewnenakhov** [reply]
well, server, web and ios versions are under a very active development (develop branch is very active as you can see), and we're not very far from them becoming a good communication product, and subsequently releasing them. Old Android version was abandoned and is being rewritten using many lessons we learned while developing iOS and Web versions. It'll probably take some time to get done. 

It was indeed removed from Google Play by Google (they needed some changes to our privacy policy) and we decided not to fix this issue for the time being - Xabber Android is still available from the F-droid anyway.
